<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>450</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>JDM FW Update v1.1</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QMainWindow
{
     border-image: url(:/lauout/Image/bg.png);
}</string>
  </property>
  <property name="toolButtonStyle">
   <enum>Qt::ToolButtonIconOnly</enum>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="contextMenuPolicy">
    <enum>Qt::NoContextMenu</enum>
   </property>
   <widget class="QTableView" name="tableView">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>494</width>
      <height>220</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_Smart">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>330</y>
      <width>91</width>
      <height>26</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
     border-image: url(:/lauout/Image/button.png);
	Color: white
}</string>
    </property>
    <property name="text">
     <string>SMART</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_re">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>310</y>
      <width>41</width>
      <height>39</height>
     </rect>
    </property>
    <property name="accessibleDescription">
     <string/>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
     border-image: url(:/lauout/Image/refresh.png);
}

QPushButton:hover
{
      border-image:url(:lauout/Image/refresh_hover.png);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_Title">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>511</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Sans Serif</family>
      <pointsize>14</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/lauout/Image/redbar.png);
color:white</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_info">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>310</y>
      <width>41</width>
      <height>39</height>
     </rect>
    </property>
    <property name="accessibleDescription">
     <string/>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
     border-image: url(:/lauout/Image/about.png);
}

QPushButton:hover
{
     border-image: url(:/lauout/Image/about_hover.png);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_min">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>310</y>
      <width>41</width>
      <height>39</height>
     </rect>
    </property>
    <property name="accessibleDescription">
     <string/>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
     border-image: url(:/lauout/Image/minimize.png);
}

QPushButton:hover
{
     border-image: url(:/lauout/Image/minimize_hover.png);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_close">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>310</y>
      <width>41</width>
      <height>39</height>
     </rect>
    </property>
    <property name="accessibleDescription">
     <string/>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
     border-image: url(:/lauout/Image/close.png);
}

QPushButton:hover
{
     border-image: url(:/lauout/Image/close_hover.png);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>600</width>
     <height>20</height>
    </rect>
   </property>
   <property name="nativeMenuBar">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
